routers/web/explore/repo.go

```diff
@@ -166,6 +166,8 @@ func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
 // Repos render explore repositories page
 func Repos(ctx *context.Context) {
 	ctx.Data["UsersIsDisabled"] = setting.Service.Explore.DisableUsersPage
+	ctx.Data["OrganizationsIsDisabled"] = setting.Service.Explore.DisableOrganizationsPage
+	ctx.Data["CodeIsDisabled"] = setting.Service.Explore.DisableCodePage
```

Comment:
I can image two meaning of them: 
- code/org feature is disabled
- code/org tab in explore is disabled